### ISSUE TITLE
fork setup work to be done

### ISSUE THREAD

**Issue by Fishrock123** [ayojs/ayo#16]
Here's a couple things off the top of my head. I'm certainly willing to help do some guidance on any but I have limited experience in some of these areas. Digging though the commits back to January 2015 in the related files folders will surly turn up related reference-able commits for the io.js days.

- [ ] Make `Vcbuild.bat` build `/ayo.exe`.
  - I think @refack is on this: https://github.com/ayojs/ayo/pull/10#issuecomment-324719098
- [ ] Update the macOS installer to install `ayo` and a `node` symlink.
  - https://github.com/ayojs/ayo/tree/master/tools/osx-pkg.pmdoc
- [ ] Update the Windows installer to install `ayo` (and possibly a `node` symlink.)
  - https://github.com/ayojs/ayo/tree/master/tools/msvs
- [ ] Possibly update key portions of the docs / meta-capitals-documents.

... more to be added

**Comment by evanlucas** [comment]
Any ideas on release infrastructure?

**Comment by Fishrock123** [comment]
@evanlucas probably target a much smaller array so as to make it somewhat feasible, e.g. macOS 64, Ubuntu Linux, Windows 64.

Not sure if it is possible to use travis for the first two, maybe?

**Comment by addaleax** [comment]
> Not sure if it is possible to use travis for the first two, maybe?

It should be, yes. I just excluded macOS from the Travis matrix because our builds take a looong time and there are generally fewer VMs available – feel free to experiment, tho :)

**Comment by valpackett** [comment]
> target a much smaller array so as to make it somewhat feasible

Understandable, but I'd love to see more cross-platform support, not less :)

I've set up ayo on my personal buildbot cluster to make unofficial builds. [Here](https://unrelentingbuildbot.s3.dualstack.eu-west-1.amazonaws.com/index.html) you can grab one for FreeBSD/amd64!

OpenBSD currently fails with `../deps/v8/src/trap-handler/trap-handler.h:68:8: error: thread-local storage is not supported for the current target` :(

**Comment by Fishrock123** [comment]
Hmmm yeah, I do remember something about compiling on OpenBSD I think. You may want to ask that on the Node issue tracker for now if you are comfortable with doing so.

**Comment by ghost** [comment]
Sorry for stupid question. But node symlink would conflict with existing nodejs installation? I want to keep on my pc both ayo and nodejs

**Comment by valpackett** [comment]
@mememaestro you might want to use a version manager like [asdf](https://github.com/asdf-vm/asdf) to switch between different versions and forks of node, as well as other languages. (But honestly I haven't found that necessary anymore, I just use the latest and greatest all the time.)

**Comment by ghost** [comment]
@myfreeweb thank you =)

**Comment by Fishrock123** [comment]
@mememaestro Not necessarily, the symlink could just be removed or there could be an option to not have it installed from the installer. I think io.js did the latter option too however I am very much not familiar with msi or pkg installer manifests.

**Comment by ghost** [comment]
@Fishrock123 that's great
